src/libtiled/templategroup.cpp

```diff
@@ -66,9 +74,18 @@ void TemplateGroup::addTileset(const SharedTileset &tileset)
     if (mTilesets.contains(tileset))
         return;
 
-    // TODO: If a tileset was  not used by any  template it will be added again
+    // TODO: If a tileset was not used by any template it will be added again
```

Comment:
Probably, at some point, unused tilesets in a template group were not registered, so saving a template that uses an unused tileset would think that the tileset was new, so the tileset was added again, but currently this case causes no problems, so the todo should be removed.